src/pynwb/tests/unit/test_ernwbfile2.py

```diff
@@ -76,6 +76,15 @@ def test_add_ref(self):
         self.assertEqual(len(nwbfile.external_resources.entities), 2)
 
         self.assertEqual(nwbfile.external_resources.objects[0],
-                         (container.object_id, 'unit'))
+                         (container.object_id, 'TimeSeries/data/unit', ''))
         self.assertEqual(nwbfile.external_resources.objects[1],
-                         (table.object_id, 'test_col'))
+                         (table.columns[0].object_id, '', ''))
```

Comment:
Interacting with columns by name is generally preferred over by index.

```suggestion
                         (table['test_col'].object_id, '', ''))
```